src/app/content/utils.ts

```diff
@@ -9,6 +9,30 @@ export { getBookPageUrlAndParams, getPageIdFromUrlParam, getUrlParamForPageId, t
 export { stripIdVersion } from './utils/idUtils';
 export { scrollSidebarSectionIntoView } from './utils/domUtils';
 
+export const cleanArchiveResponse = (archiveContent: any) => {
+  const chapterSlug = /^chapter-/;
+  const appendixSlug = /^appendix-/;
+  const chapterTitle = 'Chapter';
+  const appendixTitle = 'Appendix';
+
+  if (archiveContent.tree) {
+    archiveContent.tree.contents.map((item: any) => {
```

Comment:
if you change the type above you can probably just remove this, but if not the correct type is `ArchiveTree | ArchiveTreeSection`